packages/office-ui-fabric-react/src/components/SpinButton/examples/SpinButton.Stateful.Example.tsx

```diff
@@ -9,11 +9,13 @@ export class SpinButtonStatefulExample extends React.Component<any, any> {
     return (
       <div style={{ width: '400px' }}>
         <SpinButton
-          label="SpinButton with custom implementation:"
+          label={'SpinButton with custom implementation:'}
+          min={0}
+          max={100}
           value={'7' + suffix}
           onValidate={(value: string) => {
             value = this._removeSuffix(value, suffix);
-            if (value.trim().length === 0 || isNaN(+value)) {
+            if (Number(value) > 100 || Number(value) < 0 || value.trim().length === 0 || isNaN(+value)) {
```

Comment:
This feels a little weird; if you type in 1000, it goes back to 0 units. I would expect it to revert back to the old value. Is there a way to do this?

Reply:
I was just matching the behavior of all the other examples... this whole page, on all the SpinButtons, upon entering a number out of range, the value reverts to 0 and not the most recent value. Like on Basic SpinButton, if you first change 0 to 20 and then put -40 or 8592, it goes to 0, not 20.